src/I18nBundle/EventListener/Frontend/ResponseExceptionListener.php

```diff
@@ -228,4 +233,24 @@ private function enablePimcoreContext(): void
         DataObject\AbstractObject::setGetInheritedValues(true);
         DataObject\Localizedfield::setGetFallbackValues(true);
     }
+
+    protected function logToHttpErrorLog(Request $request, $statusCode)
+    {
+        $uri = $request->getUri();
+        $exists = $this->db->fetchOne('SELECT date FROM http_error_log WHERE uri = ?', [$uri]);
+        if ($exists) {
+            $this->db->executeQuery('UPDATE http_error_log SET `count` = `count` + 1, date = ? WHERE uri = ?', [time(), $uri]);
+        } else {
+            $this->db->insert('http_error_log', [
+                'uri' => $uri,
+                'code' => (int) $statusCode,
+                'parametersGet' => serialize($_GET),
+                'parametersPost' => serialize($_POST),
+                'cookies' => serialize($_COOKIE),
+                'serverVars' => serialize($_SERVER),
```

Comment:
Don't let sloppy pimcore developers influence you! :) Please use sanitized data only (like we did in v3):

https://github.com/dachcom-digital/pimcore-i18n/blob/3c443a69d0c15b20365b6e4db06309d7ca03f5da/src/I18nBundle/EventListener/Frontend/ResponseExceptionListener.php#L325-L328